### ISSUE TITLE
Quality of ScandiQA-sv dataset

### ISSUE THREAD

**Issue by marma** [ScandEval/ScandEval#437]
Hi!

First: I can no longer find Scandeval/ScandiQA-sv-mini on HF, so I'm instead referring to alexandrainst/scandi-qa. If these two datasets are not the same, then please ignore the following.

I am evaluating the results of models for QA and wanted to check the results from the model. From what I can tell by an, admittedly small, sample there are a lot of errors in the dataset. I took the first 250 QA pairs and checked them manually. The result was that a lot of the "errors" made by the model were actually errors in the dataset to the point that the model had more correct answers than the answer column.

Some of these may come from translation problems such as "Vem sjöng I Want to be with You Everywhere?" with the answer Fleetwood Mac. The correct answer given the context is "Christine MacVie". Potentially "who played" is translated as "vem sjöng" (who sang) given rise to this confusion?

There are numerous other examples, for example the date of the Super Bowl XXXIV which is reported as 1999 when it was in fact played in 2000.

Again, please ignore this if these errors have been weeded out in the dataset used by ScandEval.

**Comment by saattrupdan** [comment]
Hey!

Thanks for raising this issue. 

We have indeed noticed that the quality of the dataset is not fantastic, but thanks for giving us even more evidence. These samples are mere (human) translations of a subset of Google's Natural Questions, so most of the factual errors are from that original source. 

The dataset used in ScandEval is generated by [this script](https://github.com/ScandEval/ScandEval/blob/main/src/scripts/create_scandiqa.py), and doesn't (necessarily) remove these bad samples. 

As there aren't any alternative extractive QA datasets for Swedish (as well as Danish and Faroese), we're currently working on a GPT-4 generated (and thus non-commercial) QA dataset for these languages, which will replace ScandiQA.

**Comment by saattrupdan** [comment]
Closing due to no activity. If you don't feel like my answer was sufficient then please re-open the issue 🙂

**Comment by marma** [comment]
Sorry for the slow response :) Closing this issue is fine

I, understand. I am looking forward to the new dataset it will be interesting to test out since what I am actually doing is evaluating a synthetic QA training dataset generated using Mixtral 8x7B by finetuning multiple models. That dataset is not very good for testing since it often repeats the question in the answer.

Which brings me to a suggestion: the metric for QA. Since repeating the question in the answer is common for generative models, it would be great to get recall as a metric as well. Precision will vary wildly when the models is chatty.
